pkg/trainer/replicas.go

```diff
@@ -218,20 +200,20 @@ func (s *TFReplicaSet) Create(config *tfv1alpha1.ControllerConfig) error {
 			})
 		}
 
-		log.Infof("Creating Job: %v", newJ.ObjectMeta.Name)
-		createdJob, err := s.ClientSet.BatchV1().Jobs(s.Job.job.ObjectMeta.Namespace).Create(newJ)
+		log.Infof("Creating Pod: %v", newP.ObjectMeta.Name)
+		createdPod, err := s.ClientSet.CoreV1().Pods(s.Job.job.ObjectMeta.Namespace).Create(newP)
 
-		// If the job already exists do nothing.
+		// If the Pod already exists do nothing.
 		if err != nil {
 			if k8s_errors.IsAlreadyExists(err) {
-				log.Infof("%v already exists.", s.jobName(index))
+				log.Infof("%v already exists.", s.genName(index))
 
 			} else {
 				s.recorder.Eventf(s.Job.job, v1.EventTypeWarning, FailedCreateReason, "Error creating: %v", err)
-				return k8sErrors.NewAggregate([]error{fmt.Errorf("Creating Job %v returned error.", createdJob.ObjectMeta.Name), err})
+				return k8sErrors.NewAggregate([]error{fmt.Errorf("Creating Pod %v returned error.", createdPod.ObjectMeta.Name), err})
 			}
 		} else {
-			s.recorder.Eventf(s.Job.job, v1.EventTypeNormal, SuccessfulCreateReason, "Created job: %v", createdJob.Name)
+			s.recorder.Eventf(s.Job.job, v1.EventTypeNormal, SuccessfulCreateReason, "Created Pod: %v", createdPod.Name)
 		}
 	}
 	return nil
```

Comment:
https://github.com/tensorflow/k8s/pull/344/files#diff-d2bc8c1807fa25d2b911d2d781f48a07R236

```go
err = s.ClientSet.BatchV1().Jobs(s.Job.job.ObjectMeta.Namespace).DeleteCollection(&meta_v1.DeleteOptions{}, options)
```

I think we should update here, too.

Reply:
Yes, thanks, done.